ui/src/app/shared/edge/edgeconfig.ts

```diff
@@ -413,7 +412,7 @@ export class EdgeConfig {
      * Lists all available Factories, grouped by category.
      */
     public static listAvailableFactories(factories: { [id: string]: EdgeConfig.Factory }): CategorizedFactories[] {
-        const allFactories = [
+        const allFactories: CategorizedFactories[] = [
             {
                 category: { title: 'Simulatoren', icon: 'flask-outline' },
                 factories: Object.entries(factories)
```

Comment:
allFactories.factories was mixing flat([]) list and nested([[]]). this is not type first. I rewrite flat-style because this is easily expected from the name `factories`.